src/decapodes_it.jl

```diff
@@ -0,0 +1,272 @@
+module ASKEMDecapodes
+
+export ASKEMDecaExpr, ASKEMDecapode, ASKEMDeca, SummationDecapode, parse_decapode
+
+# using ..SyntacticModelsBase
+using ..AMR
+
+using StructTypes
+# using Decapodes
+using MLStyle
+
+using ACSets
+using ACSets.InterTypes
+
+using Reexport
+@reexport using MLStyle
+@reexport using ACSets
+using ACSets.ADTs
+using ACSets.ACSetInterface
+
+import Unicode
+
+using ..AMR.amr
+
+@intertypes "decapodes.it" module decapodes
+  import ..amr
+end
+
+using .decapodes
+
+
+normalize_unicode(s::String) = Unicode.normalize(s, compose=true, stable=true, chartransform=Unicode.julia_chartransform)
+normalize_unicode(s::Symbol)  = Symbol(normalize_unicode(String(s)))
+DerivOp = Symbol("∂ₜ")
+append_dot(s::Symbol) = Symbol(string(s)*'\U0307')
+
+term(s::Symbol) = decapodes.Var(normalize_unicode(s))
+term(s::Number) = decapodes.Lit(Symbol(s))
+
+term(expr::Expr) = begin
+    @match expr begin
+        #TODO: Would we want ∂ₜ to be used with general expressions or just Vars?
+        Expr(:call, :∂ₜ, b) => decapodes.Tan(decapodes.Var(b)) 
+        Expr(:call, :dt, b) => decapodes.Tan(decapodes.Var(b)) 
+
+        Expr(:call, Expr(:call, :∘, a...), b) => decapodes.AppCirc1(a, term(b))
+        Expr(:call, a, b) => decapodes.App1(a, term(b))
+
+        Expr(:call, :+, xs...) => decapodes.Plus(term.(xs))
+        Expr(:call, f, x, y) => decapodes.App2(f, term(x), term(y))
+
+        # TODO: Will later be converted to Op2's or schema has to be changed to include multiplication
+        Expr(:call, :*, xs...) => decapodes.Mult(term.(xs))
+
+        x => error("Cannot construct term from  $x")
+    end
+end
+
+function parse_decapode(expr::Expr)
```

Comment:
Should SynacticModels be doing decapodes operations like `parse_decapode`? What do you think the scope boundary is?